src/main/java/org/openstreetmap/atlas/checks/flag/CheckFlag.java

```diff
@@ -485,6 +485,10 @@ public Task getMapRouletteTask()
                     .map(flaggedRelation -> flaggedRelation.asGeoJsonFeature(this.identifier))
                     .forEach(features::add);
         }
+        if (!this.fixSuggestions.isEmpty())
+        {
+            task.setCooperativeWork(this.fixSuggestions);
+        }
```

Comment:
I think this should be optional. Can we add a parameter to this method to set whether we want to include the fix suggestions? This could also be accompanied by an overloaded version of this method that enables this by default, to keep things backwards compatible. We should then also have a command argument for the upload command to set this.

Reply:
Yeah 100% agreed. Updates are in 1f1d8c1